src/ReadConfig.cxx

```diff
@@ -21,6 +21,9 @@
 
 #include <stdlib.h>
 #include <string.h>
+#include <filesystem>
+#include <fstream>
```

Comment:
`<iostreams>` shall not be used

Reply:
Okay, I will use C style file handling then.